### DISCUSSION TITLE
Option to drag only on long press

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [PuruVJ/neodrag#139] [Ideas]
Add option to drag only on long press like 500ms.
Specially in mobile, I want the user be able to scroll the window without dragging the item unintentionally.

**Comment by PuruVJ** [comment]
I was working on a `delay` property, got sidetracked. PRs welcome!

**Reply by juliomuhlbauer** [reply]
Do you have some initial draft of the PR or how do you would implement?

**Reply by juliomuhlbauer** [reply]
This https://github.com/PuruVJ/neodrag/pull/134 wouldn't work?

**Reply by PuruVJ** [reply]
Afraid not, that one has a different purpose. `delay` is what you'd need

**Reply by juliomuhlbauer** [reply]
Do you have some initial draft of the delay pr?
